### ISSUE TITLE
statistics about dataset

### ISSUE THREAD

**Issue by xiaxin1998** [hanjialiang/DeepRec#1]
Thanks for sharing codes for DeepRec.
Would you mind share the statistics of preprocessed dataset Taobao? After preprocessing, how many users and items and how many sessions in training set and test set? How is the average length of sessions?

**Comment by hanjialiang** [comment]
Please refer to our paper in https://dl.acm.org/doi/10.1145/3442381.3449942.
